examples/java8/src/main/java/org/apache/beam/examples/complete/game/GameStats.java

```diff
@@ -93,12 +90,6 @@
  */
 public class GameStats extends LeaderBoard {
 
-  private static final String TIMESTAMP_ATTRIBUTE = "timestamp_ms";
-
-  private static DateTimeFormatter fmt =
-      DateTimeFormat.forPattern("yyyy-MM-dd HH:mm:ss.SSS")
-          .withZone(DateTimeZone.forTimeZone(TimeZone.getTimeZone("PST")));
-
   /**
    * Filter out all but those users with a high clickrate, which we will consider as 'spammy' uesrs.
    * We do this by finding the mean total score per user, then using that information as a side
```

Comment:
1. "Filter out all users but those.." 2. What is a 'side'?  3.Add period at the end.